evmcore/tx_cacher.go

```
@@ -62,7 +62,9 @@ func newTxSenderCacher(threads int) *txSenderCacher {
 func (cacher *txSenderCacher) cache() {
 	for task := range cacher.tasks {
 		for i := 0; i < len(task.txs); i += task.inc {
-			types.Sender(task.signer, task.txs[i])
+			// this error is ignored because we want to cache the sender if valid,
+			// and if not, it simply not cached
```

Sender is cached inside transactions the first time it is queried.
Sender may fail if the signature verification fails, in this case the value will not be cached, any subsequent call to Sender will retrieve the same error, therefore it is safe to ignore errors here.